[docs][no ci] Added VSCode extension to installation (#13078)
expo/expo: docs/pages/get-started/installation.md

### Recommended Tools

- [VSCode Editor](https://code.visualstudio.com/download)

>   - [VSCode Expo Extension](https://marketplace.visualstudio.com/items?itemName=byCedric.vscode-expo) for app.json debugging and autocomplete.

Reviewer: ```suggestion
  - [VSCode Expo Extension](https://marketplace.visualstudio.com/items?itemName=byCedric.vscode-expo) for `app.json` debugging and autocomplete.
```

>   - [VSCode Expo Extension](https://marketplace.visualstudio.com/items?itemName=byCedric.vscode-expo) for app.json debugging and auto complete.

Reviewer: I think we should link to a docs page about configuring visual studio code for expo development from here rather than inlining a link to expo extension. useful to also suggest the react native extension imo

Reviewer: we can add this later

>   - [VSCode Expo Extension](https://marketplace.visualstudio.com/items?itemName=byCedric.vscode-expo) for app.json debugging and auto complete.

Reviewer: ```suggestion
  - [VSCode Expo Extension](https://marketplace.visualstudio.com/items?itemName=byCedric.vscode-expo) for app.json debugging and autocomplete.
```

>   - [VSCode Expo Extension](https://marketplace.visualstudio.com/items?itemName=byCedric.vscode-expo) for app.json debugging and auto complete.

Reviewer: ```suggestion
  - [VSCode Expo Extension](https://marketplace.visualstudio.com/items?itemName=byCedric.vscode-expo) for `app.json` debugging and auto complete.
```

- [Yarn](https://classic.yarnpkg.com/en/docs/install)

> - Windows users: [PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows) or Bash via WSL. The VSCode terminal is also very good on windows computers.

Reviewer: ```suggestion
- Windows users: [PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows), Bash via WSL, or the VSCode terminal.
```
